### DISCUSSION TITLE
Unbridged camera.. how?!

### DISCUSSION THREAD

**Discussion by wingnut1979** [Sunoo/homebridge-camera-ffmpeg#1078] [Q&A]
OK I'm throwing in the towel.
I can't for the life of me figure out how to get the camera added while unbridged.

My config is below.
If I change the camera to bridged mode, it works fine - but then I get messages about how it's gonna be all slow.
I have plans to add several more cameras in the near future so I'd like to avoid unneeded slowness.

Will buy a pizza for anyone who can help me here!

{
    "bridge": {
        "name": "Homebridge 0E4E",
        "username": "0E:53:43:76:0E:4E",
        "port": 51874,
        "pin": "949-73-827"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "f",
            "lang": "auto",
            "platform": "config"
        },
        {
            "name": "Camera FFmpeg",
            "cameras": [
                {
                    "name": "test1",
                    "manufacturer": "Reolink",
                    "unbridge": true,
                    "videoConfig": {
                        "source": "-i rtsp://user:pass@address//h264Preview_01_sub",
                        "maxStreams": 6,
                        "maxWidth": 0,
                        "maxHeight": 0,
                        "maxBitrate": 0,
                        "audio": true
                    }
                }
            ],
            "platform": "Camera-ffmpeg"
        }
    ]
}

**Comment by wingnut1979** [comment]
update:
I added a line:
"stillImageSource": "-i http://address/cgi-bin/api.cgi?cmd=Snap&channel=0&user=user&password=password",

Now I no longer get the spam messages about stuff being slow.
Obviously I'm still bridged... but does it matter now?

**Reply by Sunoo** [reply]
stillImageSource often helps response times significantly, since otherwise it has to connect to the video stream and grab the first frame.

I still recommend unbridge mode, but I need to figure out why some users have been having trouble pairing with it. The usual fix is, if you’re running Config UI X, to remove the cameras with the “Unpair External Accessories” option under Homebridge Settings. I am looking into this though, as that seems to be required at times it shouldn’t be.

**Reply by wingnut1979** [reply]
OK I never did get the unbridged thing figured out..
But the rest of my cameras arrived!
I'm running this as my DVR software: https://www.ispyconnect.com/
Doing 640x480 streamed and 4096x3072 recorded on 25 cameras.
Simultaneously, I have all of them ported into Homekit via Homebridge and ffmpeg.
All this running on a 2018 Mac Mini dumping the data files into my NAS box.
I'd say you've earned your pizza sir.

Edit:  I was going to use the "tip" function... but the button seems to do nothing...
How can I get you pizza money?!

**Reply by Sunoo** [reply]
There should be links on the main page on the right hand side. There are a few options, but most people seem to use Ko-fi https://ko-fi.com/sunookitsune

And, for the record, the next version should have fixes to hopefully resolve the problems people have been hitting with unbridge, just not sure when I’ll have a chance to get that released.
